src/simulator.zig

```diff
@@ -171,6 +172,14 @@ fn chance(random: *std.rand.Random, p: u8) bool {
     return random.uintAtMost(u8, 100) <= p;
 }
 
+//returns the next argument for the simulator or null (if none avail)
```

Comment:
```suggestion
/// Returns the next argument for the simulator or null (if none available)
```

Reply:
For a function doc comment, we should use `///` and then also use full words e.g. `available` rather than `avail`.